doc/man7/openssl-quic.pod

```diff
@@ -7,15 +7,17 @@ openssl-quic - OpenSSL QUIC
 =head1 DESCRIPTION
 
 OpenSSL 3.2 and later features support for the QUIC transport protocol.
-Currently, only client connectivity is supported. This man page describes the
-usage of QUIC client functionality for both existing and new applications.
+Initially, only client connectivity was supported. Now, server connectivity is
+also supported. This man page describes the usage of QUIC functionality for both
+existing and new applications.
```

Comment:
I would modify this piece of text. It will very quickly become out of date and irrelevant. In a couple of years time, who cares that we launched with only client support? I don't think we need to mention it.

```suggestion
You can use OpenSSL's QUIC capabilities for both client and server applications.
This man page describes the usage of QUIC functionality for both existing and
new applications.
```